src/js/tech/html5.js

```diff
@@ -139,8 +137,6 @@ class Html5 extends Tech {
           el.appendChild(trackEl);
         }
       }
-
-      Lib.insertFirst(el, player.el());
```

Comment:
Yes I am aware of that. It only insert the element if it is not already in the player:

```javascript
// Add the tech element in the DOM if it was not already there
if (this.tech.el().parentNode != this.el()) {
  Lib.insertFirst(this.tech.el(), this.el());
}
```